Fireworks/Core/interface/FWViewType.h

```diff
@@ -46,22 +46,11 @@ class FWViewType {
     kTypeSize
   };
 
-  enum EBit {
-    k3DBit = 1 << k3D,
-    kRhoPhiBit = 1 << kRhoPhi,
-    kRhoZBit = 1 << kRhoZ,
-    kRhoPhiPFBit = 1 << kRhoPhiPF,
-    kISpyBit = 1 << kISpy,
-    kLegoBit = 1 << kLego,
-    kLegoHFBit = 1 << kLegoHF,
-    kLegoPFECALBit = 1 << kLegoPFECAL,
-    kGlimpseBit = 1 << kGlimpse,
-    kTableBit = 1 << kTable,
-    kTableHLTBit = 1 << kTableHLT,
-    kTableL1Bit = 1 << kTableL1,
-    kGeometryBit = 1 << kGeometryTable,
-    kOverlapBit = 1 << kOverlapTable
-  };
+  static constexpr int k3DBit = 1 << k3D, kRhoPhiBit = 1 << kRhoPhi, kRhoZBit = 1 << kRhoZ,
```

Comment:
Please use `unsigned int` (behavior of bit operations is easiest to understand on unsigned)

Reply:
done